main.go

```diff
@@ -52,76 +55,120 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var kubeAPIServer string
+	var kubeconfig string
+	var watchNamespace string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&kubeAPIServer, "kubernetes", "", "The Kubernetes API URL, used for in-cluster client construction.")
+	flag.StringVar(&kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file.")
+	flag.StringVar(&watchNamespace, "namespace", "", "Namespace that the controller watches to reconcile objects.")
+
 	opts := zap.Options{
 		Development: true,
```

Comment:
Do we always want this set to dev?

Reply:
Nope. There's no reason, really, to use development mode unless you're leveraging `DPanic`.

Reply:
Oh i had just left it like that in case it would be useful while debugging during initial development 👍 
We can remove it.